config/default.js

```diff
@@ -320,6 +321,8 @@ module.exports = {
 
   enableFeatureInlineAddonReview: true,
 
+  enableNewHomeHero: false,
```

Comment:
I was thinking I would turn this on in the dev configs once the guide pages are set up

Reply:
Shouldn't we add the pages first then?

Also, a feature flag must start with `enableFeature`.

Reply:
>Shouldn't we add the pages first then?

I am working on the other pages but wanted to tackle this page first. I can create another issue to turn this "on" in dev once those are ready. Would that be okay?


I've updated the feature flag name, thanks : )

Reply:
ok, that sounds good.

(also, that looks super shiny)